src/granite_orm.cr

```diff
@@ -48,6 +48,31 @@ class Granite::ORM
     {% SETTINGS[:timestamps] = true %}
   end
 
+  # retrive the parent relationship
+  macro belongs_to(model_name)
+    {% FIELDS["#{model_name.id}_id".id.symbolize] = Int64 %}
```

Comment:
`field {{model_name.id}}_id : Int64`?

Reply:
Ah, good point. I seem to have gotten lost in macro land :P